src/NLog/LayoutRenderers/Wrappers/CachedLayoutRendererWrapper.cs

```diff
@@ -47,7 +47,18 @@ namespace NLog.LayoutRenderers.Wrappers
     [ThreadAgnostic]
     public sealed class CachedLayoutRendererWrapper : WrapperLayoutRendererBase
     {
-        private string cachedValue;
+        /// <summary>
+        /// A value indicating when the cache is cleared.
+        /// </summary>
+        public enum ClearCacheOption 
```

Comment:
Look good, but I think a `[flags]` enum would be more future proof? Although i'm not sure if a flags-enum is working from the XML config. 

What do you think?